ipyleaflet/velocity.py

```diff
@@ -67,20 +67,16 @@ class Velocity(Layer):
     min_velocity = Float(0).tag(sync=True, o=True)
     max_velocity = Float(10).tag(sync=True, o=True)
     velocity_scale = Float(0.005).tag(sync=True, o=True)
-    color_scale = List([
-        "rgb(36,104, 180)",
-        "rgb(60,157, 194)",
-        "rgb(128,205,193)",
-        "rgb(151,218,168)",
-        "rgb(198,231,181)",
-        "rgb(238,247,217)",
-        "rgb(255,238,159)",
-        "rgb(252,217,125)",
-        "rgb(255,182,100)",
-        "rgb(252,150,75)",
-        "rgb(250,112,52)",
-        "rgb(245,64,32)",
-        "rgb(237,45,28)",
-        "rgb(220,24,32)",
-        "rgb(180,0,35)"
-    ]).tag(sync=True, o=True)
+    colormap = Any(linear.YlOrRd_04)
+    color_scale = List([]).tag(sync=True, o=True)
+
+    @default('color_scale')
+    def _default_color_scale(self):
+        self.color_scale = []
+
+        for color in self.colormap.colors:
+            rgb_tuple = tuple(int(x * 256) for x in color[:3])
+            rgb_str = f"rgb{rgb_tuple}"
+            self.color_scale.append(rgb_str)
+
+        return self.color_scale
```

Comment:
By default we don't have the same color scale as before, maybe we should?
Also, it might be good to check that we don't pass a `colormap` *and* a `color_scale`.

Reply:
Yes, you are right that we don't have the same default `color scale`as before. Here we propose that this colormap is defined from the colormap in such a way that there is a total correspondance between the mapping of colors in the colormap and the mapping of colors on the map. To avoid any confusion, we propose that the user only has to give a colormap and we make the color_scale a private attribute (change proposed in [ed48e77](https://github.com/jupyter-widgets/ipyleaflet/pull/1035/commits/ed48e7785aa650a471cf0f199051c28ed4b260fc) )